lib/controllers/issueish-detail-controller.js

```diff
@@ -213,7 +224,12 @@ export default createFragmentContainer(BareIssueishDetailController, {
       commitCount: {type: "Int!"},
       commitCursor: {type: "String"},
     ) {
-      ...issueishDetailView_repository
+      ...issueDetailView_repository
```

Comment:
todo: figure out a way to not duplicate this fragment, because it seems like there should be a way to avoid it.

Reply:
Interesting.  This is what I had originally tried but I got a relay compiler error. But now it's working.